packages/@adobe/spectrum-css-temp/components/button/index.css

```diff
@@ -116,6 +116,7 @@ governing permissions and limitations under the License.
     bottom: 0;
     top: 0;
     margin: calc(var(--spectrum-alias-focus-ring-gap) * -1);
+    transform: translateX(0);
```

Comment:
Can you add a comment on these lines to explain why this is there?

Reply:
sure, added here https://github.com/adobe-private/react-spectrum-v3/pull/582